### ISSUE TITLE
Where to Store 'Corner' Units

### ISSUE THREAD

**Issue by GoFroggyRun** [open-source-economics/Tax-Calculator#753]
In #752 and #555, we have encountered a couple 'counter intuitive' units that behave oddly. It seems to me that we might want to have a place to store these units, as well as other helpful information (such as tax year, w/o extrapolation/blowup and etc) in order to replicate the desired case. 

Moreover, we might also want to build some simple tests accordingly, based on different cases that those 'corner' units would result in.

**Comment by martinholmer** [comment]
After investigating the "corner" unit discussed in #752, my opinion is that maintaining a catalogue of unusual tax filing units is a very low priority project.  There are many other projects related to Tax-Calculator that would have a much higher return on staff effort.

But maybe I'm missing something.  What do others think about this proposed project?

@MattHJensen @feenberg @Amy-Xu @GoFroggyRun @zrisher

**Comment by MattHJensen** [comment]
I've been thinking of this issue, #753, and @GoFroggyRun's accompanying pull request, #764, as an attempt to document oddities we've found in the tax code. We wouldn't be storing corner units for the sake of storing corner units, rather, we would be using corner units to illustrate the oddities. 

Sean brings up a good example of one of these oddities by referencing #555. There we encountered some really high MTRs that were a result of social security benefits being added to AGI. In that discussion, the best way for me to explain and communicate this phenomenon to others was by highlighting a single simple tax record: https://github.com/open-source-economics/Tax-Calculator/issues/555#issuecomment-176400567. 

Keeping track of oddities in the tax code that lead to counterintuitive tax-calculator results seems useful to me as they are often the cause of bug false alarms, which are stressful and time consuming. Tax-Calculator results that confuse us may also indicate areas where tax policy could be improved. These oddities could serve as the subject matter for future short vignettes about weird stuff in the tax code. 

All of that said, I am also not sure where the right place to document these tax code oddities is, or what information should be included.
